### PR TITLE
Ignore certain paths

### PR DESCRIPTION
**Pull request by jangie**
For our use case, we sometimes have .js files which are intended for
the client side. Oftentimes, these are minimized (and are yet huge),
and this needlessly slows the scan/replace down.

### CODE REVIEW

**Comment by soldair** [comment]
looks great!

File: bin/cli.js [hunk 1]
```diff
@@ -81,6 +99,11 @@ function rewrite(cb){
   walkdir(dir,function(p,stat){
     if(p.substr(p.length-3) !== '.js') return
     if(~p.indexOf('.git') || ~p.indexOf('node_modules')) return
+    for (var i = 0; i < ignorePaths.length; i++){
```
Comment (soldair) [inline bin/cli.js:102]: this could be `if(ignorePaths.indexOf(p) === 0) return;` instead of the for right?

File: bin/cli.js [hunk 2]
```diff
@@ -44,6 +48,20 @@ var scope = argv.scope
 var modules = argv._
 var dir = path.resolve(process.cwd(),argv.dir)
 
+var ignorePaths = [];
+if (argv.ignorePaths.length !== 0){
+  ignorePaths = argv.ignorePaths.split(",");
+  for (var i = 0; i < ignorePaths.length; i++){
+    if (ignorePaths[i].length){
+      if (ignorePaths[i].charAt(0) !== '/'){
+        ignorePaths[i] = dir + '/' + ignorePaths[i];
```
Comment (soldair) [inline bin/cli.js:57]: i prefer `path.join`  instead of `+'/'+` to be cross platform compatible. ill update this after merge.

**Comment by soldair** [comment]
1.0.2

File: bin/cli.js (hunk 1 above)
Comment (jangie) [inline bin/cli.js:102]: Ah, true. I think that we could do that so long as we normalized the end of the inputted paths such that they matched whatever is returned as it's being walked.

File: bin/cli.js (hunk 2 above)
Comment (jangie) [inline bin/cli.js:57]: :+1: , was only thinking about unix-y stuff. Thanks!

File: bin/cli.js (hunk 1 above)
Comment (soldair) [inline bin/cli.js:102]: Walkdir has ignore paths built in too. Anyway thanks for the fixes! Amazing

**Comment by soldair** [comment]
hey added you as a collaborator! thanks for the help
check out contributing guidelines etc in https://github.com/npm/scope-it/blob/master/CONTRIBUTING.md

**Pull request merged**